tests/foreman/ui_airgun/test_dashboard.py

```diff
@@ -138,6 +147,35 @@ def test_positive_host_configuration_chart(session):
         assert dashboard_values['chart']['No report'] == '100%'
 
 
+@tier2
+def test_positive_new_host(session):
+    """Check if the New Hosts widget is working in the Dashboard UI
+
+    :id: 123fadc6-e3e0-4a49-8bca-2bf672460359
+
+    :Steps:
+
+        1. Create new host in the application
+        2. Navigate to Monitor -> Dashboard
+        3. Review the New Hosts widget
+
+    :expectedresults: New Hosts widget contains information about just created host
+
+    :CaseLevel: Integration
+    """
+    org = entities.Organization().create()
+    loc = entities.Location().create()
+    host = entities.Host(organization=org, location=loc).create()
+    with session:
+        session.organization.select(org_name=org.name)
+        session.location.select(loc_name=loc.name)
+        dashboard_values = session.dashboard.read('NewHosts')['hosts']
+        assert len(dashboard_values) == 1
```

Comment:
@oshtaier I am having a query on assertion, any specific reason to check the length of  `dashboard_values` ?
because if we do not specify this assertion by all way it would fails on next assertion if does not able to fetch the value.

Reply:
Nope, as:
1) if we have more than one row (which is invalid) and first row is expected one, you will never know about that fact
2) if we have more than one row and first row is not one we need, you will get the failure which looks confusing right from CI page, so it is harder to understand root cause without re-running the test